as/build.xml

```diff
@@ -0,0 +1,282 @@
+<!--
```

Comment:
I don't think we would need build.xml anymore. We are not using ant/ivy.
build.gradle should be sufficient